src/components/Timer.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+
+import Button from "./Button";
+
+import useInterval from "../hooks/useInterval";
+
+import formatTime from "../utils/formatTime";
+import minutesToSeconds from "../utils/minutesToSeconds";
+
+function Timer() {
+  const [seconds, setSeconds] = useState(minutesToSeconds(0.05));
```

Comment:
Niçin dakika cinsinden parametre alan bir kısıma `0.05` gibi bir değer verdik.

Reply:
test etmek için 3 saniyeye düşürmüştüm ama son commitimde değeri tekrar 25 olarak düzelttim.